pegasus-spark-bulkloader/src/main/java/com/xiaomi/infra/pegasus/spark/bulkloader/BulkLoadInfo.java

```diff
@@ -0,0 +1,16 @@
+package com.xiaomi.infra.pegasus.spark.bulkloader;
+
+public class BulkLoadInfo {
+
+  private String cluster;
+  private int app_id;
+  private String app_name;
+  private int partition_count;
+
+  BulkLoadInfo(String cluster, String app_name, int app_id, int partition_count) {
```

Comment:
```suggestion
  BulkLoadInfo(String cluster, String appName, int appId, int partitionCount) {
```

Reply:
As above reason, the defines are all server rules